cli/src/lib/pub.ts

```diff
@@ -162,17 +163,47 @@ const pub = (options: PublishOptions) => {
 
   const pubOpts = parsePublishOptions(options)
 
-  if (options.stdin) {
+  const handleFileRead = async () => {
+    const filePath = processPath(options.fileRead!)
+    if (!filePath) {
+      signale.error('File path is required when reading from file.')
+      process.exit(1)
+    }
+
+    try {
+      signale.pending('Reading file...')
+      let bufferData = readFile(filePath)
```

Comment:
```suggestion
      const bufferData = readFile(filePath)
```

Since there is no place to change the buffer data, should we use const instead of let?